README.md

```diff
@@ -154,4 +154,20 @@ Steps you can try if the script isn't working:
 
 ## Development
 
-If you want to play with the `restream` code, you will have to [install Rust](https://www.rust-lang.org/learn/get-started) and [setup the reMarkable toolchain](https://github.com/canselcik/libremarkable#setting-up-the-toolchain) to do cross-platform development.
+If you want to play with the `restream` code, you will have to [install Rust](https://www.rust-lang.org/learn/get-started).
+
+There are two ways of building the required restream binary for streaming the reMarkable framebuffer.
+
+### Using docker and the toltec toolchain
```

Comment:
Since the build instructions are fairly shorty, I would maybe just use bullet points?

Reply:
I prefer headings, but made the change since I also don't have a strong opinion about it 😄 .